pysen_ls/__main__.py

```diff
@@ -47,8 +50,14 @@ def main() -> None:
         default=3746,
         help="Port number for the tcp server. Only works with --tcp.",
     )
+    parser.add_argument(
+        "--log-file",
+        help="redirect logs to the given file instead of writing to stderr.",
```

Comment:
```suggestion
        help="Save logs to the given file.",
```

Can I ask you to use a capital letter at the beginning of the sentence?
Also "Save" sounds better as the default handler is `NullHandler` in my suggestion.